recipes/cppunit/all/conanfile.py

```diff
@@ -0,0 +1,113 @@
+            del self.options.fPIC
+
+    def build_requirements(self):
+        if tools.os_info.is_windows and not tools.get_env("CONAN_BASH_PATH"):
+            self.build_requires("msys2/20200517")
+        if self.settings.compiler == "Visual Studio":
+            self.build_requires("automake/1.16.2")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("{}-{}".format(self.name, self.version), self._source_subfolder)
+
+    @contextmanager
+    def _build_context(self):
+        if self.settings.compiler == "Visual Studio":
+            with tools.vcvars(self.settings):
+                env = {
+                    "AR": "{} lib".format(tools.unix_path(self.deps_user_info["automake"].ar_lib)),
+                    "CC": "{} cl -nologo".format(tools.unix_path(self.deps_user_info["automake"].compile)),
+                    "CXX": "{} cl -nologo".format(tools.unix_path(self.deps_user_info["automake"].compile)),
+                    "NM": "dumpbin -symbols",
+                    "OBJDUMP": ":",
+                    "RANLIB": ":",
+                    "STRIP": ":",
+                }
+                with tools.environment_append(env):
+                    yield
+        else:
+            yield
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
+        if self.settings.compiler == "Visual Studio":
+            self._autotools.flags.append("-FS")
+            self._autotools.cxx_flags.append("-EHsc")
+        yes_no = lambda v: "yes" if v else "no"
+        conf_args = [
+            "--enable-shared={}".format(yes_no(self.options.shared)),
+            "--enable-static={}".format(yes_no(not self.options.shared)),
+            "--enable-debug={}".format(yes_no(self.settings.build_type == "Debug")),
+            "--enable-doxygen=no",
+            "--enable-dot=no",
+            "--enable-werror=no",
+            "--enable-html-docs=no",
+        ]
+        self._autotools.configure(args=conf_args, configure_dir=self._source_subfolder)
+        return self._autotools
+
+    def build(self):
+        with self._build_context():
+            autotools = self._configure_autotools()
+            autotools.make()
+
+    def package(self):
+        self.copy("COPYING", src=self._source_subfolder, dst="licenses")
+        with self._build_context():
+            autotools = self._configure_autotools()
+            autotools.install()
+
+        os.unlink(os.path.join(self.package_folder, "lib", "libcppunit.la"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+
+    def package_info(self):
+        libsuffix = (".dll" if self.settings.os == "Windows" and self.options.shared else "") + (".lib" if self.settings.compiler == "Visual Studio" else "")
```

Comment:
with MSVC and shared `libsuffix =".dll.lib` is that correct?

Reply:
Yes, I tested this on MSVC.
If I remember correct, in general, when libtool is used to build shared libraries,
it generates a `.dll.lib` file for MSVC and a `.dll.a` file for mingw.

Reply:
I suggest to rename import lib suffix if visual studio.

Reply:
I would advice to rename import lib suffix if visual studio (.dll.lib => .lib)